libwallet/assets_config.go

```diff
@@ -278,3 +278,15 @@ func (mgr *AssetsManager) SetTotalBalanceVisibility(data bool) {
 func genKey(prefix, identifier interface{}) string {
 	return fmt.Sprintf("%v-%v", prefix, identifier)
 }
+
+// IsDexFirstVisit checks if its the users first visit to DCRDEX page
+func (mgr *AssetsManager) IsDexFirstVisit() bool {
+	var data bool
+	mgr.db.ReadWalletConfigValue(sharedW.DexFirstVisitKey, &data)
+	return data
+}
+
+// SetDexFirstVisit sets the first visit to DCRDEX .
+func (mgr *AssetsManager) SetDexFirstVisit(data bool) {
+	mgr.db.SaveWalletConfigValue(sharedW.DexFirstVisitKey, data)
+}
```

Comment:
Let's remove these config stuffs. We won't be needing them. Instead, a future PR will instantiate the DEX client core object and check if the user has set a DEX trading password. Until the user sets a DEX trading password, we'll keep showing the onboarding page.